src/clients/ayast_setup.rb

```diff
@@ -126,6 +126,13 @@ def Setup
       end
       WFM.CallFunction("inst_autoconfigure", [])
 
+      # Restarting autoyast-initscripts.service in order to run
+      # init-scripts in the installed system.
+      cmd = "systemctl restart autoyast-initscripts.service"
+      Builtins.y2milestone("executing:%1", cmd)
+      ret = SCR.Execute(path(".target.bash_output"), cmd)
+      Builtins.y2milestone("calling ret:%1", ret)
```

Comment:
Why not merging both y2milestone calls? On the other hand, we should replace Builtins.y2milestone by log.info.